EIPS/eip-4762.md

```diff
@@ -251,6 +236,13 @@ When a **write** event of `(address, sub_key, leaf_key)` occurs, perform the fol
 
 Note that tree keys can no longer be emptied: only the values `0...2**256-1` can be written to a tree key, and 0 is distinct from `None`. Once a tree key is changed from `None` to not-`None`, it can never go back to `None`.
 
+Note that values should only be added to the witness if there is sufficient gas to cover their associated event costs.
+
+`CREATE*` and `*CALL` reserve 1/64th of the gas before the nested execution. In order to match the behavior of this charge with the pre-fork behavior of access lists: 
+
+ * this 1/64th of the gas is subtracted **AFTER** charging the witness costs when performing a `CALL`, `CODECALL`, `DELEGATECALL` or`STATICCALL`
```

Comment:
@gballet, I wonder if we're missing an extra clarification here.

The `*CALL` variants that can be value-bearing, should also respect this rule?

Today in geth, we have this order (e.g: `makeCallVariantGasEIP4762(gasCall)`):
- Charge general `*CALL` costs. (`makeCallVariantGasEIP4762(...)`)
- Do the 1/64 stuff ((`gasCall(...)`)
- Charge BALANCE if `transfersValue` (`gasCall(...)`)

So I'm wondering if this requires a clarification, or was intentional. If was intentional it means that today in kaustinenv6-geth we need a change.